src/ripple/test/jtx/Env.h

```diff
@@ -420,6 +423,11 @@ class Env
     std::shared_ptr<STObject const>
     meta();
 
+    /** Execute a client command */
+    template <class T, class... Args>
+    std::pair<int, Json::Value>
+    rpc (T const& t, Args const&... args);
```

Comment:
`T` does not need to be a separate paramter. `template<class... Args> is all that's needed.